nfts.proto

```diff
@@ -223,11 +223,11 @@ message NftEvents {
     CreateEditionTransaction polygon_retry_drop = 23;
     MintEditionTransaction polygon_retry_mint_drop = 24;
     TransferPolygonAsset polygon_transfer_asset = 25;
-    MetaplexMasterEditionTransaction solana_create_drop = 26;
-    MetaplexMasterEditionTransaction solana_retry_drop = 27;
-    MetaplexMasterEditionTransaction solana_update_drop = 28;
-    MintMetaplexEditionTransaction solana_mint_drop = 29;
-    MintMetaplexEditionTransaction solana_retry_mint_drop = 30;
```

Comment:
You can't delete events schema registry will reject the update so you'll need to leave them and just add on the replacements.

Reply:
We can rename the event name if the data type is same

Reply:
I think its forward compatible but not backward. I'll add the replacements